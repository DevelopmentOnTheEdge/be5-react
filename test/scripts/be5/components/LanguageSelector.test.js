import React    from 'react';
import renderer from 'react-test-renderer';
import LanguageSelector     from  '../../../../src/scripts/be5/components/LanguageSelector';
import be5 from '../../../../src/scripts/be5/be5';

it('snapshot menu', () => {

  be5.net.request = function (path, attr, callback) {
    callback({"languages":["en"],"messages":{"no":"нет","yes":"да","fio":"Ф.И.О."},"selected":"en"})
  };

  const component = renderer.create(
    <LanguageSelector />
  );

  expect(component.toJSON()).toMatchSnapshot();
});