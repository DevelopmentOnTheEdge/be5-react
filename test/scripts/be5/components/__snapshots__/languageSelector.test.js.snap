// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot menu 1`] = `
<div
  className="languageBox"
>
  <div
    className="languageList"
  >
    <div
      className="language selectedLanguage"
    >
      en
    </div>
  </div>
</div>
`;
